test/e2e/helpers.js

```diff
@@ -760,36 +760,9 @@ const sendScreenToConfirmScreen = async (
   await openActionMenuAndStartSendFlow(driver);
   await driver.fill('[data-testid="ens-input"]', recipientAddress);
   await driver.fill('.unit-input__input', quantity);
-  if (process.env.MULTICHAIN) {
-    // check if element exists and click it
-    await driver
-      .findElement({
-        text: 'I understand',
-        tag: 'button',
-      })
-      .then(
-        (_found) => {
-          driver.clickElement({
-            text: 'I understand',
-            tag: 'button',
-          });
-        },
-        (error) => {
-          console.error('Element not found.', error);
-        },
-      );
 
-    await driver.clickElement({
-      text: 'Continue',
-      tag: 'button',
-    });
-  } else {
-    await driver.clickElement({
-      text: 'Next',
-      tag: 'button',
-      css: '[data-testid="page-container-footer-next"]',
-    });
-  }
+  await driver.clickElement({ text: 'Continue', tag: 'button' });
+  await driver.clickElement({ text: 'Confirm', tag: 'button' });
```

Comment:
Remove clicking on the confirm button. We do not want to submit the transaction. The `sendScreenToConfirmScreen` helper is used for ppom tests, and on those tests we use mainnet (for ppom support.. we mock all network requests)